src/category_theory/groupoid.lean

```diff
@@ -5,6 +5,26 @@ Authors: Reid Barton, Scott Morrison, David Wärn
 -/
 import category_theory.full_subcategory
 
+/-!
+# Groupoids
```

Comment:
Could this explain at least succinctly what is a groupoid here? Btw, do we have algebraic groupoids in mathlib? If so, a pointer would be great.

Reply:
The definition is immediately below: "asserting that all morphisms have inverses".

Reply:
Okay sure. It's not the clearest but I'll take it.

Reply:
I'm genuinely unsure how it could be any clearer. :-)